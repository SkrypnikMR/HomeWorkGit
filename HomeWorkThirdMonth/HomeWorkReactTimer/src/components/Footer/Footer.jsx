import React, { Component } from 'react';
import './Footer.scss';

class Footer extends Component {
    render() {
        return (
            <header className="footer">
                <h3>
                    React Timer made by:
                    <a href="https://t.me/Skripnik2201" target="_blank" rel="noreferrer">@Skripnik2201</a>
                </h3>
            </header>
        );
    }
}

export default Footer;
