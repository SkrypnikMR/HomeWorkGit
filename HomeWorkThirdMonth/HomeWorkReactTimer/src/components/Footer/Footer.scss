.footer{
    display: flex;
    justify-content: flex-end;
    align-items: center ;
    color: white;
    background-color: #5C55AE;
    font-size: 15px;
    h3{
        padding-right: 20px;
        a{
            text-decoration: none;
            color: rgb(229, 208, 237);
            font-style:italic;
        }
    }
}