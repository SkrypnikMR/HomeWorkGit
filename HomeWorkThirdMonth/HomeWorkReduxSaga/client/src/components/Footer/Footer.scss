.footer{
    margin-top: auto ;
    width: 100%;
    display: flex;
    bottom: 0px;
    justify-content: flex-end;
    align-items: center ;
    color: white;
    background-color: #f5da0b;
    font-size: 15px;
    h3{
        padding-right: 20px;
        a{
            text-decoration: none;
            color: rgb(229, 208, 237);
            font-style:italic;
        }
    }
}