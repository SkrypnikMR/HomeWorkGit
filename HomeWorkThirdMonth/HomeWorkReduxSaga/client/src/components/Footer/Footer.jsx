import React from 'react';
import './Footer.scss';

const Footer = () => {
    return (
        <footer className="footer">
            <h3>
                Movies made by:
                <a href="https://t.me/Skripnik2201" target="_blank" rel="noreferrer">@Skripnik2201</a>
            </h3>
        </footer>
    );
}

export default Footer;
