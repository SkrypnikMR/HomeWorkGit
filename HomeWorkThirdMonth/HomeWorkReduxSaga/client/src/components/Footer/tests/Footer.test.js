import React from 'react';
import { shallow, mount } from 'enzyme';
import Footer from '../Footer';

describe('Footer', () => {
    it('Should match snapshot', () => {
        const component = shallow(<Footer />);
        expect(component.html()).toMatchSnapshot();
    });
    it('Should have h3', () => {
        const component = mount(<Footer />);
        expect(component.find('h3')).toHaveLength(1);
    });
    it('Should have a', () => {
        const component = mount(<Footer />);
        expect(component.find('a')).toHaveLength(1);
    });
});
