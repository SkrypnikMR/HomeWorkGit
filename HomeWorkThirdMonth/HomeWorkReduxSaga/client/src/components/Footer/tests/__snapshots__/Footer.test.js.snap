// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should match snapshot 1`] = `"<footer class=\\"footer\\"><h3>Movies made by:<a href=\\"https://t.me/Skripnik2201\\" target=\\"_blank\\" rel=\\"noreferrer\\">@Skripnik2201</a></h3></footer>"`;
